paper.bib

```diff
@@ -43,5 +43,5 @@ @article{Alaux2025EUWLCPathways
   author={Nicolas Alaux and Nicolas Bechstedt and Xiaoyang Zhong and Alessio Mastrucci and Delphine Ramon and Dominik Steinberger-Maierhofer and Karen Allacker and Alexander Passer and Martin R{\"o}ck},
   journal={Forthcoming},
   year={2025},
-  url={TBD} # Can we have a preprint DOI? At least for revision/final version

```

Comment:
@mroeck is this fine if I remove this comment? It prevents the paper from compiling